pkg/baseline/packageIndex.json

```diff
@@ -1,127 +1,133 @@
 {
-  "Packages": {
```

Comment:
Please don't change the indentation of this file, because it looks like this whole file is changed.